src/elasticsearch/user-elasticsearch.service.ts

```
@@ -769,6 +771,19 @@ export class UserElasticsearchService implements OnModuleInit {
               return;
             }
 
+            // Handle auto_tags filtering
+            if (field === 'auto_tags') {
```

ESLint

[error] Replace `'auto_tags'` with `"auto_tags"`

(prettier/prettier)